docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,740 @@
+    "Graph.RunUefi"
+    "Graph.BootstrapWinPE"
+    "Graph.Write.Quanta.BIOS.NVRAM"
+    "Graph.Arista.Zerotouch.vEOS"
+
+Or review the list of all the built-in tasks available to be used in workflows
+
+- ``curl http://localhost/api/2.0/workflows/tasks | python -m json.tool``
+
+.. code-block:: JSON
+
+    [
+      {
+        "friendlyName": "Boot LiveCD",
+        "injectableName": "Task.Os.Boot.LiveCD",
+        "implementsTask": "Task.Base.Os.Install",
+        "options": {
+          "profile": "boot-livecd.ipxe",
+          "completionUri": "renasar-ansible.pub",
+          "version": "livecd",
+          "repo": "{{api.server}}/LiveCD/{{options.version}}"
+        },
+        "properties": {
+          "os": {
+            "linux": {
+              "distribution": "livecd"
+            }
+          }
+        }
+      },
+      ...
+
+Adding a simulated server
+---------------------------
+
+The Vagrantfile included in the example setup includes a reference to simulated
+server provide by the `InfraSim`_ project. You can download and boot this simulated
+server, which includes an interface to IPMI as well as simulates the physical machine
+with an internal VM.
+
+.. _InfraSim: http://infrasim.readthedocs.io
+
+By default, RackHD will PXE boot this instance, interrogate it, and then leave it alone.
+
+- Set up the simulated server
+
+.. code::
+
+    vagrant up quanta_d51
+
+This command will start up vagrant with the GUI console available. You can see
+the Quanta d51 control with the vBMC quanta simulator by using VNC to connect
+to 127.0.0.1:15901 (or 127.0.0.1 display 10001). You can log into the VM hosting
+this simulation with the default credentials of username ``root``, and password ``root``.
+
+The IPMI credentials that it is providing on ``closednet`` use the username ``admin``
+and password ``admin``.
+
+Once the node has been discovered by RackHD, you can see it through the API.
+
+- ``curl http://localhost:9090/api/2.0/nodes | python -m json.tool``
+
+.. code-block:: JSON
+
+    [
+        {
+            "autoDiscover": "false",
+            "id": "57967193a045ba7c0800207b",
+            "identifiers": [],
+            "name": "Enclosure Node QTFCJ05160195",
+            "obms": [],
+            "tags": [],
+            "type": "enclosure"
+        },
+        {
+            "autoDiscover": "false",
+            "id": "579680825d434579084ff910",
+            "identifiers": [
+                "52:54:be:ef:aa:ee"
+            ],
+            "name": "52:54:be:ef:aa:ee",
+            "obms": [],
+            "sku": null,
+            "tags": [],
+            "type": "compute"
+        }
+    ]
+
+Viewing the geneaology
+---------------------------
+
+You can view all of the information collected about a specific node through the
+``catalogs`` URI. For the example above, using the node with the ID **579680825d434579084ff910**:
+
+- ``curl http://localhost:9090/api/2.0/nodes/579680825d434579084ff910/catalogs | jq``
```

Comment:
A newer version of `jq` resolves this. The latest is 1.5 (and what I apparently just installed when I started into this docs work) - not sure if this changed in 1.4 or 1.5